### PR TITLE
questions added 

### PR DESCRIPTION
**Pull request by apurvabhandari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | S.No. |Question |
 |---|---|
 | 1 | Will the Notice period mentioned in the Offer letter, can be increased in future(Actually happened to me)? |
+| 2 | What is in-hand salary that will get from gross CTC? |
+| 3 | Is this position for 24x7 rotational? |
+| 4 | Leave policy |
+| 5 | If start-up, then ask for fund management |
+| 6 | Medical insurance or extra benefits apart from mentioned in offer letter |
```
Comment (sanjeevyadavIT) [inline README.md]: Hi @apurvabhandari could you please frame these points as a question

**Review by sanjeevyadavIT — COMMENTED** [review]
Thank you for your contribution


Frame the points in question mode

**Review by apurvabhandari — COMMENTED** [review]
updated

**Comment by sanjeevyadavIT** [comment]
Thank you Apurva, please share this repo with others and star if you like it :smile:

**Comment by sanjeevyadavIT** [comment]
Hi @apurvabhandari could you please explain what this questions means
> Is this position for 24x7 rotational?

**Pull request merged**